universal-login-example/src/services/IdentityService.js

```diff
@@ -25,20 +25,19 @@ class IdentityService {
       this.identity.address
     );
     const {address} = new Wallet(this.privateKey);
+    const filter = {contractAddress: this.identity.address, key: address};
     this.subscription = this.sdk.subscribe(
```

Comment:
How about a oneliner?
`this.subscription = this.sdk.subscribe('KeyAdded', filter, () => {`

Reply:
✅